deps/jerry-v8/src/v8jerry_utils.cpp

```diff
@@ -34,6 +34,33 @@ jerry_value_t JerryPolyfill::BuildMethod(const char* name, const char* fn_args,
     return method;
 }
 
+static jerry_value_t JerryHandlerStackTrace(const jerry_value_t func,
+                                            const jerry_value_t this_val,
+                                            const jerry_value_t args_p[],
+                                            const jerry_length_t args_count)
+{
+    if (!jerry_is_feature_enabled(JERRY_FEATURE_LINE_INFO) || args_count == 0 || !jerry_value_is_object(args_p[0]))
+    {
+        printf("Line info is disabled or the argument is not an object.\n");
+        return jerry_create_undefined();
+    }
+
+    jerry_value_t stack_string = jerry_create_string((const jerry_char_t*)"stack");
+    jerry_value_t stack_trace = jerry_get_backtrace(0);
+    jerry_value_t set_result = jerry_set_property (args_p[0], stack_string, stack_trace);
```

Comment:
AFAIK, there should be no space(s) before the opening parenthesis.

Reply:
Fixed.